src/components/ExpandableGroup.stories.tsx

```diff
@@ -25,38 +29,65 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 const Content = ({ accordion }: { accordion: boolean | undefined }) => {
-  const linkPath = `/?path=/story/components-expandablegroup--${accordion ? 'accordion' : 'default'}`
-
-  return <div>
-    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Neque ipsa
-    voluptatibus soluta nobis unde quisquam temporibus magnam debitis
-    quidem. Ducimus ratione corporis nesciunt earum vel est quaerat
-    blanditiis dolore ipsa?&nbsp;
-    <a href={linkPath}>
-      Lorem link
-    </a>
-  </div>
-}
+  const linkPath = `/?path=/story/components-expandablegroup--${
+    accordion ? 'accordion' : 'default'
+  }`;
 
+  return (
+    <div>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Neque ipsa
+      voluptatibus soluta nobis unde quisquam temporibus magnam debitis quidem.
+      Ducimus ratione corporis nesciunt earum vel est quaerat blanditiis dolore
+      ipsa?&nbsp;
+      <a href={linkPath}>Lorem link</a>
+    </div>
+  );
+};
 
 export const Default: Story = {
-  render: args => <ExpandableGroup {...args}>
-    {[1, 2, 3].map(val =>
-      <Expandable header={`Expandable ${val}`} inAccordion={args.accordion}>
-        <Content accordion={args.accordion} />
-      </Expandable>
-    )}
-  </ExpandableGroup>,
+  render: args => (
+    <ExpandableGroup {...args}>
+      {[1, 2, 3].map(val => (
+        <Expandable header={`Expandable ${val}`} inAccordion={args.accordion}>
+          <Content accordion={args.accordion} />
+        </Expandable>
+      ))}
+    </ExpandableGroup>
+  ),
+  play: async ({ args, canvasElement, step }) => {
+    const canvas = within(canvasElement);
+
+    const el = await canvas.findByTitle('Expandable 1');
+
+    await step('Expand tile - Aria Expanded is true', async () => {
+      await userEvent.click(el);
+      waitFor(
+        () => {
+          expect(el.ariaExpanded).toBe('true');
+        },
+        { timeout: 1000 }
+      );
+    });
```

Comment:
I see that this [test fails on the initial page load](http://localhost:6006/?path=/story/components-expandablegroup--default) but after a re-run it completes successfully.  I imagine this could be because the elements are not fully loaded when the test is initially run.  

This doesn't seem to be an issue for the [individual Expandable interaction test](http://localhost:6006/?path=/story/components-expandable--test-expand-collapse).  

Please see if you can get the tests to consistently pass on load.